udata/core/dataset/api.py

```diff
@@ -233,18 +235,19 @@ def post(self, dataset):
         return resource, 201
 
     @api.secure
-    @api.doc('reorder_resources', **common_doc)
-    @api.expect([fields.String])
+    @api.doc('update_resources', **common_doc)
+    @api.expect([resource_fields])
     @api.marshal_with(resource_fields)
     def put(self, dataset):
         '''Reorder resources'''
         DatasetEditPermission(dataset).test()
-        new_resources = []
-        for rid in request.json:
-            resource = get_by(dataset.resources, 'id', UUID(rid))
-            new_resources.append(resource)
-        dataset.resources = new_resources
-        dataset.save()
+        data = {'resources': request.json}
```

Comment:
I think we should not have to do this at the server side but directly in the client/admin. Thoughts?